src/resources/views/crud/columns/select.blade.php

```diff
@@ -1,21 +1,25 @@
 {{-- single relationships (1-1, 1-n) --}}
```

Comment:
Before this, when a `relationship` column was empty it was showing `-`. Now it's showing `` (empty), which is inconsistent with the rest of the column types. 

I believe `relationship` falls back to `select` in this case (`articles` on MonsterCrudController):

![2021-11-21 10 59 33](https://user-images.githubusercontent.com/1032474/142756027-1254d207-5fab-42f3-8f48-9725bb1e7467.gif)

Reply:
Same problem for `select_multiple` from what I see.

Reply:
Got it, it's because it is a Laravel collection, and I'm checking for `empty()`.
It's fixed now!

Reply:
Btw, previously it was using a custom variable to store the processed results, since now it's all on `$column['value']` it was causing that issue, the end result could be an empty collection, or an array.